metricbeat/docs/modules/aws/ebs.asciidoc

```diff
@@ -9,7 +9,6 @@ beta[]
 
 include::../../../../x-pack/metricbeat/module/aws/ebs/_meta/docs.asciidoc[]
 
-This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
Thanks!

Reply:
Umm, but this changes are making CI fail :thinking: I wonder what I have different in my local machine.

Reply:
`BEAT_STRICT_PERMS=false` did the trick